### PR TITLE
Fix typos and various improvements for the German document

### PR DESCRIPTION
**Pull request by mhei**
- fix typos (see: https://en.wikipedia.org/wiki/Coordinated_Universal_Time, https://de.wikipedia.org/wiki/Durchkopplung, https://www.duden.de/sprachwissen/rechtschreibregeln)
- add/improve/unify hyperlinks
- improve readability
- various unifications

[ mhei:
 - reverted Sicherheitskonzept -> Sicherungskonzept since nobody knows the correct document title
 - fixed small whitespace issues ]

This PR supersedes  PR #19 since the original author (@descodess) did not respond to the request to rework his PR.
(Closes #19)

### CODE REVIEW

File: OCMF-de.md [hunk 1]
```diff
@@ -363,13 +355,13 @@ wünschenswert. Folgende sind dazu reserviert:
...
-Zum Aufbau eines Bezugs zwischen dem zur Signaturprüfung zu benutzenden Public-Key und des
-Ladepunktes dienen eine oder mehrere Seriennummern, alternativ kann auch eine direkte Identifikation des
+Zum Aufbau eines Bezugs zwischen dem zur Signaturprüfung zu benutzenden Public-Key und dem
+Ladepunkt dienen eine oder mehrere Seriennummern, alternativ kann auch eine direkte Identifikation des
 Ladepunktes per ID erfolgen.
 
 Die Public-Keys zum Ladepunkt müssen auf anderem Wege als diesem Protokoll (Out-of-Band) an die Prüfkomponente in Verbindung mit der verwendeten ID oder den Seriennummern übermittelt werden, z.B. über ein zentrales Register.
 
-Um eine Signaturkomponente eindeutig zu identifizieren werden in diesem Protokoll folgende Merkmakle definiert:
+Um eine Signaturkomponente eindeutig zu identifizieren, werden in diesem Protokoll folgende Merkmakle definiert:
```
Comment (lategoodbye) [inline OCMF-de.md:364]: While changing this line, please fix Merkmakle to Merkmale

**Comment by FlorianRieglerKeba** [comment]
Would it be possible to check the en-version as well and add the missing links as well as updating the spelling?

File: OCMF-de.md (hunk 1 above)
Comment (mhei) [reply OCMF-de.md]: done

**Comment by mhei** [comment]
Yes, it is on my personal todo list to re-check the english version. But I would like to see this merged anyway to have a fixed/good-in-shape german version in master before we remove it and focus on the english version.

**Pull request merged**